src/main/views/et-fortnightly-list.njk

```diff
@@ -13,8 +13,7 @@
   <div class="parent-box">
 
     <h2 id="page-heading" class="govuk-heading-l">
-      {{ title1 }}: {{ region }}<br/>
-      {{ title2 }}
+      {{ title1 }} {{ title2 }}: {{ region }}
```

Comment:
@KianKwa made the same changes in his PR. He converted title1 and title2 to title. We can do same here as well.